GLib-2.0.gir

```diff
@@ -5,7 +5,7 @@ and/or use gtk-doc annotations.  -->
 <repository xmlns="http://www.gtk.org/introspection/core/1.0" xmlns:c="http://www.gtk.org/introspection/c/1.0" xmlns:glib="http://www.gtk.org/introspection/glib/1.0" version="1.2">
   <package name="glib-2.0"/>
   <c:include name="glib.h"/>
-  <namespace name="GLib" version="2.0" shared-library="libglib-2.0.so.0,libgobject-2.0.so.0" c:identifier-prefixes="G" c:symbol-prefixes="g,glib">
+  <c:include name="glib-object.h"/><namespace name="GLib" version="2.0" shared-library="libglib-2.0.so.0,libgobject-2.0.so.0" c:identifier-prefixes="G" c:symbol-prefixes="g,glib">
```

Comment:
This is wrong IMHO. It should only be needed by GObject-2.0.gir

Reply:
The problem here is that GLib-2.0.gir defines GType. Definition of GType
is also present in GLib headers, but *only* when parsed by g-ir-scanner.
This is exactly the ugly hack I mentioned in PR description, not sure
what would be best solution for this.

Reply:
Ah I see, and usually bindings don't include the headers so it's not a problem there as they get the definition from the .gir file. Nice!

Reply:
Special-casing GType might be fine though, it's a quite fundamental type for all the gir machinery :)